devito/dimension.py

```diff
@@ -226,6 +227,30 @@ class TimeDimension(Dimension):
     """
 
 
+class DefaultDimension(Dimension):
+    is_Default = True
+
+    def __new__(cls, name, **kwargs):
+        newobj = sympy.Symbol.__new__(cls, name)
+        newobj._spacing = kwargs.get('spacing', Scalar(name='h_%s' % name))
+        newobj.default_value = kwargs.get('default_value', None)
+        return newobj
+
+    def _arg_defaults(self, start=None, size=None, alias=None):
+        """
```

Comment:
drop docstring, as if identical to the superclass', then it should just be inherited. I'm now trying to follow this approach to avoid proliferation of redundant docstrings

Reply:
Fixed